packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -47,6 +48,17 @@ const configurations = {
       addressesToRemove: ['agoric1w8wktaur4zf8qmmtn3n7x3r0jhsjkjntcm3u6h'],
     },
   },
+  EMERYNET: {
+    committeeName: 'Economic Committee',
+    voterAddresses: {
+      gov1: 'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce',
+      gov2: 'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang',
+    },
+    highPrioritySendersConfig: {
+      addressesToAdd: [],
+      addressesToRemove: [],
+    },
+  },
```

Comment:
Shouldn't we use a similar pattern to other test configs? i.e. allow gov1, gov2 and gov4 to be voterAddresses - removing gov3 and adding gov4 in the high priority sender's config. 

Because I believe without this, we won't really be testing _replacement_ of committee members. Thoughts?

Reply:
@mujahidkay Per my [Slack message](https://agoricopco.slack.com/archives/C07QD1HR0VA/p1730327103212909?thread_ts=1730209146.912099&cid=C07QD1HR0VA) just now - that is the EC change testing config, not the Oracle address testing config. They are different things but there is an overlap between Gov 1 and Gov 2 accounts. Gov 1 and Gov2 are both oracles and EC members

Reply:
That is correct. I was confirming it for [this change](https://github.com/Agoric/agoric-sdk/pull/10376#discussion_r1823516304)